nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/resources/OSGI-INF/secured-comment-manager-override.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.comment.manager.override.property">
```

Comment:
Shouldn't be:
```suggestion
<component name="org.nuxeo.comment.manager.override.secured">
```
or maybe:
```suggestion
<component name="org.nuxeo.comment.manager.override.tree">
```
?

Reply:
you are right, thx 👍  go for first suggestion